proposed/2021/OpenReadMeMarkDownFiles.md

```diff
@@ -0,0 +1,38 @@
+# Automatically Open `README.md` Files When Installing a NuGet Package in Visual Studio Spec
+
+* Status: **In Review**
+* Author(s): [David Cueva Cortez](https://github.com/dcc7497)
+* Issue: [10353](https://github.com/NuGet/Home/issues/10353) Feature Request: Automatically Open `readme.md` Files
+* Type: Feature
+
+## Problem Background
+
+When developers install packages from Visual Studio&#39;s package manger and if that package has a README file, Visual Studio will automatically open a `README.txt` file. However, Visual Studio does not have a way of opening a rendered `README.md` file, just a text version of the README. While the README allows for the developer to get more information about the package, having a rendered markdown file open automatically allows for information to be displayed in an organized manner. When rendered, the markdown file has links one can follow, different sections can be clearly marked with headers, among other things. It provides the same information as a txt file with the added benefit of being rendered in an easier to follow format.
```

Comment:
If you install Castle.Core in PM UI, it will show you a Readme file. What's different with this new experience?

![image](https://user-images.githubusercontent.com/1192347/124653473-30a90580-de52-11eb-93c5-ecaa1f700431.png)

Reply:
For this feature we want to do more than just have open a plain text file. We want to have it open a rendered markdown, something to add more style than just text. This is a rough idea of what it should look like once all said and done
![image](https://user-images.githubusercontent.com/63263324/124658043-25f16f00-de58-11eb-8e96-eb60a19f2608.png)

Reply:
This wirefrime is great! Could you please add it to the spec?